employee-rostering-frontend/src/store/contract/reducers.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ActionType, ContractList, ContractAction } from './types';
+import {withElement, withoutElement, withUpdatedElement} from 'util/ImmutableCollectionOperations';
```

Comment:
Would it be possible to use the spacing consistently? For destructuring (`const { name, age } = this.props.person;`), object literals (`const person = { name: "Joe", age: 5 }`) and ES6 imports, spaces around the curly braces are preferred.

Reply:
Didn't run linter yet as PR is a WIP; the linter might fix it

Reply:
True. Let's fix #247 first.